plugins/node/instrumentation-tedious/test/api.ts

```diff
@@ -16,17 +16,31 @@
 
 import * as assert from 'assert';
 import { promisify } from 'util';
-import type { Connection, Request, TYPES, ConnectionConfig } from 'tedious';
+import * as semver from 'semver';
+
+import type {
+  Connection,
+  Request,
+  TYPES,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ConnectionConfig as ConnectionConfigLegacy,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ConnectionConfiguration as ConnectionConfigNew,
+} from 'tedious';
```

Comment:
Since we have both types from `tedious` and from the still existing `@types/tedious` dependency, is this import able to join them both?

Reply:
I am not aware of any other solution. Please suggest a working alternative approach :)

Reply:
I was actually wondering if you know how it works. Since it's test code, i don't think we need to worry and the current implementation is fine with me, but curious how typescript handles situation where the types exists twice in `node_modules`

Reply:
If the tests are running, depending on the **current installed** tedious version (types are either in tedious or in @types), the code imports whats available. 

I feel like TS was not designed to fulfill such a use case.